apps/web/app/constants.ts

```diff
@@ -352,3 +352,39 @@ export const statusOptions = [
 ];
 
 export const DEFAULT_WORK_HOURS_PER_DAY = 8;
+
+// 20+ Major currencies
```

Comment:
@CREDO23 why we don't call API that will return list currencies from backend API?